schema_utils.py

```diff
@@ -221,6 +240,20 @@ class _Validators(object):
     """
     @classmethod
     def get(cls, validator_id):
+        """Gets the validation method corresponding to the given validator ID.
+
+        Args:
+            validator_id: str. The validator ID.
+
+        Returns:
+            *. Return the value of the named attribute of object.
```

Comment:
I don't understand this docstring. I don't see a "named attribute" anywhere and I don't know what "object" refers to.

Reply:
This comment still applies.